src/foremast/templates/pipeline/pipeline_kubernetes_simple.json.j2

```diff
@@ -0,0 +1,252 @@
+{
+  "expectedArtifacts": [
+    {
+      "defaultArtifact": {
+        "id": "da60c92f-e404-40c6-8ace-8f0e60bb8f0f",
```

Comment:
Spinnaker expects the ID to be set and match the ID of the artifact, but on the Foremast side it could be possible to map them before sending it.  I'm doing something similar with the canary (accept a name, map to canary ID)